pkg/handlers/authorization/handler.go

```diff
@@ -45,36 +47,25 @@ func (auth *authorizationHandler) Process(req *http.Request, context *handlers.R
 	context.Token = getBearerTokenFrom(req)
 	if context.Token == "" {
 		log.Debugf("Skipping %s as there is no bearer token present", auth.Name())
-		return req, nil
+		return req, errors.New("missing bearer token")
 	}
 	sanitizeHeaders(req)
-	json, err := auth.osClient.TokenReview(context.Token)
+	rolesProjects, err := auth.cache.getRolesAndProjects(context.Token)
 	if err != nil {
-		log.Errorf("Error fetching user info %v", err)
 		return req, err
```

Comment:
If there is any additional context in this function that you could add to the error message, you could do something like this:
```go
   return req, fmt.Errorf("Error fetching user info for user %s: %v", user.Name, err)
```